src/components/PeoplePicker/PeoplePicker.md

```diff
@@ -0,0 +1,32 @@
+### Notes for use
+
+PeoplePicker is simply NormalPeoplePicker [exported from Fabric](https://developer.microsoft.com/en-us/fabric#/components/peoplepicker).
+
+
+### Examples
+
+Basic usage:
+
+```js { "props": { "data-description": "NormalPeoplePicker example" } }
```

Comment:
Same here, I wouldn't mention `NormalPeoplePicker` in the description, cause if we change the underlying implementation we'll have to change this, and then the screenshot will have a different name and we won't be able to compare the before and after easily.

Reply:
agree, this data-description could just be "basic"